tools/diagnostics/full_bundle.py

```diff
@@ -146,6 +150,11 @@ def create(self) -> (int, "FullBundle"):
             for t in active_marathon_service.get("tasks", [])
             if is_service_scheduler_task(self.package_name, self.service_name, t)
         ]
+        if len(scheduler_tasks) == 0:
```

Comment:
`if not scheduler_tasks:` would be more Pythonic...

Reply:
Aye 82deb678bb7c08d97013021d636e2b5571ea20ae